pyMixtComp/python/setup.py

```diff
@@ -1,6 +1,16 @@
 from setuptools import setup
 
 import re
+import sys
+
+# to ensure that we error out properly for people with outdated setuptools
+# and/or pip.
+py_min_version = (3, 9)  # minimal supported python version
```

Comment:
This requirements is too restrictive, I currently use python 3.8

Reply:
My idea was to make it consistent across platforms. Since on macOS, it only works with Python3.9, we could use that as a minimum requirement for Linux and mac as well.

Reply:
I will check if it is possible to fix different minimal python version with regards to the platform